tests/testthat/test-08-endpoints-model.R

```diff
@@ -113,7 +113,7 @@ test_that("querying for an orphan task returns sensible error", {
 
   queue <- test_queue()
   id <- ids::random_id()
-  queue$queue$con$HSET(queue$queue$keys$task_status, id, "DIED")
+  queue$queue$con$HSET(r6_private(queue$queue)$keys$task_status, id, "DIED")
```

Comment:
This use of keys is tolerable as it's not in the package itself, but I wonder if this is something that we might eventually add to the testing interface of the queue somehow?